content.js

```diff
@@ -10,98 +10,60 @@ var Post = class Post {
     }
 };
 
-function loopWhole(i,len, idp, list) {
-    for (i ; i < len - 1; i++) {
-        if (document.querySelector("#pagelet_group_mall").querySelector("div").querySelectorAll("._4-u2, .mbm, ._4mrt, ._5jmm, ._5pat, ._5v3q, ._7cqq, ._4-u8")[i].getAttribute('id') === idp) {
+function getData(idp, list) {
+
+    let len = document.querySelectorAll(".userContentWrapper").length;
```

Comment:
U use `document.querySelectorAll(".userContentWrapper") ` in several places. I think U should create variable on the top function with this and U should using this variable in other.
e.g.
```
const contentWrapper = document.querySelectorAll(".userContentWrapper");
while (contentWrapper.length) {
```